app/Console/Commands/FetchTransitousLicenses.php

```diff
@@ -38,7 +38,27 @@ private function parseLicenses(): int {
         $licenses = $this->getLicenseJson();
 
         foreach ($licenses as $license) {
-            $country    = $license['region_code'];
+            $country     = $license['country_code'] ?? null;
+            $subdivision = $license['subdivision_code'] ?? null;
+            if (empty($country)) {
+                $this->error('No country code found for license: ' . json_encode($license));
+                continue;
+            }
+
+            if (!empty($subdivision)) {
+                if (!str_starts_with($subdivision, $country . '-')) {
```

Comment:
Is this a case that really happens? I think that would be a bug on the Transitous side.

Reply:
I don't really know. Just wanted to play it safe